ci/kind/kind-setup.sh

```diff
@@ -81,7 +82,8 @@ where:
   --k8s-version: specifies the Kubernetes version of the kind cluster, kind's default K8s version will be used if empty.
   --deploy-external-server: deploy a container running as an external server for the cluster.
   --all: delete all kind clusters
-  --until: delete kind clusters that have been created before the specified duration.
+  --until: delete kind clusters that have been created before the specified duration
+  --no-search-domain: specifies the custom resolv.conf file for kind cluster.
```

Comment:
This becomes more confusing. I meant using `--no-search-domain` as a flag, which will copy the system resolv.conf, remove the search option, and pass it to kind.

Reply:
It's kind of boolean type flag, like `--no-proxy` above.

Reply:
I found the easiest way to achieve it is to just set an empty dnsSearch in kind.conf:
```
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
networking:
  dnsSearch: []
```
And I feel we could perhaps doing it by default, no extra flag is needed given we never use the search domain, and it could also speed up DNS resolution.

Reply:
Sure i will do that, but I have two small doubts: 
1.  if you want me to add `--no-search-domain` as boolean type flag then that case its only use to remove search option from resolve.conf and pass it to kind. if someone wants to pass  their own  custom-resolve.conf file  to kind for testing purpose then they will not able to use it. right?

2. why we are not allowing users to pass  flag like --subnets, --ip-family... to provide their own customized resolve.conf file  to the  kind  for testing purpose.

Reply:
> I found the easiest way to achieve it is to just set an empty dnsSearch in kind.conf:
> 
> ```
> kind: Cluster
> apiVersion: kind.x-k8s.io/v1alpha4
> networking:
>   dnsSearch: []
> ```
> 
> And I feel we could perhaps doing it by default, no extra flag is needed given we never use the search domain, and it could also speed up DNS resolution.

okay, Trying.